assets/js/base/components/pagination/style.scss

```
@@ -20,6 +20,14 @@
 		padding: 0.1em 0.2em;
 		min-width: 1.6em;
 	}
+
+	// Twenty Twenty register a background color for buttons that is too specific
+	// and broad at the same time `button:not(.toggle)` so We’re Engaging in a
```

Small capitalization issue:
```suggestion
	// and broad at the same time `button:not(.toggle)` so we're engaing in a
```